### DISCUSSION TITLE
Subdomain-based mastodon instance support

### DISCUSSION THREAD

**Discussion by GeekCornerGH** [community/community#52650] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi everyone,
I'm selfhosting my own mastodon instance. Due to technical reasons, I can't run the mastodon instance on the apex domain.
My mastodon instance is setted up to generate usernames on the apex domain, meaning:
* Mastodon instance is accessible via `subdomain.example.com`
* Mastodon instance is setted up in order to allow other instance to find users like this: `@username@example.com`

I'd like to link my instance page, but I'd like github to show the __proper__ username (in this case: `@username@example.com` instead of `@username@subdomain.example.com`), would you consider this?
Thanks in advanced for your answers.

**Comment by airtower-luna** [comment]
Your post reads like you want to suggest a feature for [Mastodon](https://github.com/mastodon/mastodon), so you should probably post in the Mastodon repository. At least that's where their [roadmap page](https://joinmastodon.org/roadmap) tells you to go with feature suggestions. :wink: 

This forum is primarily for the discussion of Git and GitHub, and a discussion here is unlikely to be seen by the Mastodon developers.

**Reply by GeekCornerGH** [reply]
No, this is regarding the ability to link mastodon profile on the same place you can link twitter or linkedin.

**Reply by airtower-luna** [reply]
Ah, I see what you mean, sorry.

**Reply by 100terres** [reply]
Same issue was reported [here](https://github.com/orgs/community/discussions/55036) with more information on how all this works.

https://github.com/orgs/community/discussions/55036

**Comment by opegamanalitarok** [comment]


**Reply by GeekCornerGH** [reply]
I was talking about the place on your profile where you can add custom links.

**Reply by GeekCornerGH** [reply]
Looks like an ai generated-reply, there is no "Add custom link" button on GitHub...
